command/ssh.go

```diff
@@ -173,7 +175,7 @@ func (c *SSHCommand) Run(args []string) int {
 		// Feel free to try and remove this dependency.
 		sshpassPath, err := exec.LookPath("sshpass")
 		if err == nil {
-			sshCmdArgs = append(sshCmdArgs, []string{"-p", string(resp.Key), "ssh", "-o UserKnownHostsFile=" + userKnownHostsFile, "-o StrictHostKeyChecking=" + strictHostKeyChecking, "-p", resp.Port, username + "@" + ip.String()}...)
+			sshCmdArgs = append(sshCmdArgs, []string{"-p", string(resp.Key), "ssh", "-o UserKnownHostsFile=" + userKnownHostsFile, "-o StrictHostKeyChecking=" + strictHostKeyChecking, "-p", resp.Port, username + "@" + ip.String(), commands}...)
 			sshCmd := exec.Command(sshpassPath, sshCmdArgs...)
```

Comment:
Both here and down on line 191, can you append `commands` to `sshCmdArgs` if the length of `commands` is non-zero?  From a correctness perspective, `sshCmdArgs` should not contain an empty string element.